continuous_integration/environment-3.8-dev.yaml

```diff
@@ -60,3 +60,5 @@ dependencies:
   - python-xxhash
   - mmh3
   - pip
+  - pip:
+   - pyorc
```

Comment:
Hmm that's too bad this isn't available for Windows on conda-forge. FWIW I opened up https://github.com/conda-forge/pyorc-feedstock/issues/9 to see if that would be easy to add

Reply:
I'm curious why we're only adding this to `continuous_integration/environment-3.8-dev.yaml`. I think this particular environment file is only used in our `upstream` CI build. We might try adding it to the non-`dev` environment files instead

Reply:
Hmm - not sure when the environment-3.8.yaml change was lost.

Reply:
It looks like it may be best to avoid using pyorc for testing (It seems to crash on macos for both 3.8 and 3.9 ci).  Fortunately, my initial understanding that the existing `orc_files` fixture was supplying single-stripe orc files was completely wrong.  So, it looks like we don't really need pyorc anyway... Sorry about that.